pkgs/applications/audio/cider/default.nix

```diff
@@ -0,0 +1,30 @@
+{ appimageTools, lib, fetchurl }:
+
+appimageTools.wrapType2 rec {
+  pname = "cider";
+  version = "1.3.1308";
+
+  src = fetchurl {
+    url = "https://1308-429851205-gh.circle-artifacts.com/0/%7E/Cider/dist/artifacts/Cider-${version}.AppImage";
+    sha256 = "1lbyvn1c8155p039qfzx7jwad7km073phkmrzjm0w3ahdpwz3wgi";
+  };
+
+  extraInstallCommands =
+    in ''
+    let contents = appimageTools.extract { inherit src; };
```

Comment:
```suggestion
  extraInstallCommands = let
    contents = appimageTools.extract { inherit src; };
  in ''
```

Reply:
:facepalm: Sorry about that. Thanks for the catch.